nni/algorithms/compression/pytorch/pruning/amc/amc_pruner.py

```diff
@@ -207,7 +205,10 @@ def __init__(
 
 
     def compress(self):
+        _previous_cudnn_deterministic = torch.backends.cudnn.deterministic
+        torch.backends.cudnn.deterministic = True
```

Comment:
Why do we need to set the deterministic here, to reproduce the number in the paper(I guess)? Can we just remove the deterministic?

@chech-ms

Reply:
I check with @chicm-ms , and this line can be removed.

@mstczuo thanks for your finding, could you please remove this line directly?

Reply:
OK. new PR here: #4161